src/handlers/list.ts

```diff
@@ -9,24 +9,34 @@ export namespace ListHandler {
   const taskData = "data/db.json";
 
   export const handle = async (args: Args): Promise<void> => {
-    let taskObjects: Array<Task>;
+    let allTasks: Array<Task>;
     try {
       const data = await readFile(taskData);
       const stringContent = data.toString();
-      taskObjects = JSON.parse(stringContent);
+      allTasks = JSON.parse(stringContent);
     } catch (e: unknown) {
       console.error("Error! We couldn't read the file...");
       return;
     }
```

Comment:
We may want to move this to a separate function so that it can be reused

Reply:
Yes that sounds better! Thank you for your comment and I will do it!